impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -242,6 +241,9 @@ struct RenderPassData {
   gl.Disable(GL_CULL_FACE);
   gl.Disable(GL_BLEND);
   gl.ColorMask(GL_TRUE, GL_TRUE, GL_TRUE, GL_TRUE);
+  gl.DepthMask(GL_TRUE);
+  gl.StencilMaskSeparate(GL_FRONT, 0xFFFFFFFF);
+  gl.StencilMaskSeparate(GL_BACK, 0xFFFFFFFF);
```

Comment:
When depth buffer clipping + StC is enabled, the depth mask toggles very frequently, so we need to make sure it's on before clearing.

We've yet to utilize the stencil mask in the 2D renderer, which is why we haven't noticed this problem up until now.